.palantir/revapi.yml

```diff
@@ -829,6 +829,12 @@ acceptedBreaks:
     - code: "java.class.removed"
       old: "interface org.apache.iceberg.actions.RewritePositionDeleteStrategy"
       justification: "Removing deprecated code"
+    - code: "java.method.numberOfParametersChanged"
+      old: "method org.apache.iceberg.rest.responses.LoadTableResponse org.apache.iceberg.rest.CatalogHandlers::loadTable(org.apache.iceberg.catalog.Catalog,\
+        \ org.apache.iceberg.catalog.TableIdentifier)"
+      new: "method org.apache.iceberg.rest.responses.LoadTableResponse org.apache.iceberg.rest.CatalogHandlers::loadTable(org.apache.iceberg.catalog.Catalog,\
+        \ org.apache.iceberg.catalog.TableIdentifier, org.apache.iceberg.rest.RESTSessionCatalog.SnapshotMode)"
+      justification: "add snapshot mode when load Table"
```

Comment:
we don't want to break any existing APIs. See also https://github.com/apache/iceberg/blob/master/CONTRIBUTING.md#semantic-versioning